torch/_dynamo/variables/functions.py

```diff
@@ -381,7 +381,10 @@ def __init__(
         if closure is None:
             closure_scope = None
         self.closure_scope = closure_scope
-        self.wraps_source = wraps_source
+        # Either a source or a VT with .can_reconstruct() == True
+        self.wrapped_reconstructible: Optional[
```

Comment:
Can this type hint be moved to the `__init__` definition, or does it need to be here as well?